formats/compact-refract.md

```diff
@@ -0,0 +1,80 @@
+# Compact Refract
+
+Compact Refract is a serialization format Refract for the purpose of removing a
+lot of the object keys that are repeated throughout the full serialization of
+Refract, as seen in the Refract specification. It also allows for expressing
+structures in a tuple, which resembles other formats like XML or Lisp.
+
+## Dependencies
+
+- [Refract Base
+Specification](https://github.com/refractproject/refract-spec/blob/master/refract-spec.md)
```

Comment:
My editor splits it automatically 80 characters. I can change back.
On Sun, Jan 17, 2016 at 8:32 AM Z <notifications@github.com> wrote:

> In formats/compact-refract.md
> <https://github.com/refractproject/refract-spec/pull/53#discussion_r49947003>
> :
>
> > @@ -0,0 +1,80 @@
> > +# Compact Refract
> > +
> > +Compact Refract is a serialization format Refract for the purpose of removing a
> > +lot of the object keys that are repeated throughout the full serialization of
> > +Refract, as seen in the Refract specification. It also allows for expressing
> > +structures in a tuple, which resembles other formats like XML or Lisp.
> > +
> > +## Dependencies
> > +
> > +- [Refract Base
> > +Specification](https://github.com/refractproject/refract-spec/blob/master/refract-spec.md)
>
> 80 cols per a line I guess
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/refractproject/refract-spec/pull/53/files#r49947003>.
>